### PR TITLE
[show] Add 'features' subcommand to display status for optional features

### PR DESCRIPTION
**Pull request by pra-moh**
**- What I did**
Added 'show features' command to show status of an optional feature
**- How I did it**

**- How to verify it**
run 'show features'
```
Feature    Status
---------  --------
telemetry  enabled
```

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

-->

### CODE REVIEW

**Comment by jleveque** [comment]
Retest this please

File: show/main.py [hunk 1]
```diff
@@ -2146,5 +2146,21 @@ def tablelize(keys, data, enable_table_keys, prefix):
...
+#
+
+@cli.command('features')
+def features():
+    """Show status of optional features"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    header = ['Feature', 'Status']
+    body = []
+    status_data = config_db.get_table('FEATURES')
```
Comment (jleveque) [inline show/main.py]: Naming convention for tables in SONiC DB is singular form. Can you change this to `FEATURE`?

Comment (pra-moh) [reply show/main.py]: updated

**Comment by hui-ma** [comment]
@jleveque Pleease help check the build error and suggest how to redo the check

**Comment by jleveque** [comment]
Retest this please

**Comment by jleveque** [comment]
Retest this please

**Comment by jleveque** [comment]
Retest this please

**Comment by jleveque** [comment]
Retest this please

**Comment by jleveque** [comment]
Retest this please

**Pull request merged**